superset-frontend/plugins/legacy-plugin-chart-heatmap/src/Heatmap.js

```diff
@@ -121,6 +121,7 @@ function Heatmap(element, props) {
     let longestY = 1;
 
     records.forEach(datum => {
+      if (typeof datum.y === 'number') pixelsPerCharY = 7;
```

Comment:
can the user change the font size? Will it work in that case?
Please add a comment above explaining why this is adjusted like so

Reply:
Soon, we may be dealing with different fonts, tool. Is this an issue with the ECharts library itself that we're just working around here? If so, I wonder if they've already addressed it in a newer version.

Reply:
The user can't change the font size of y label.
I think that this maybe the issue of legacy charts library itself.

Reply:
Actually It is implemented using d3 and the implementation does not seem elegant.  Let's migrate to echarts later.

Reply:
We will later, but we need to get that prioritized on the roadmap... it'll be a while.